src/extensions/extra/PDFView/styles/viewer-overrides.css

```diff
@@ -47,6 +46,10 @@ body {
     background-repeat: no-repeat;
 }
 
+#PDFBug .stats {
+font-family: "Courier New", Courier, monospace !important;
```

Comment:
Small nit, could you please indent this line to match the rest of the stylesheet?

Reply:
Ahhh good eye @flukeout Just fixed it